src/App.js

```diff
@@ -53,24 +57,32 @@ class Landing extends React.Component {
   }
 }
 
-function SignIn() {
+class SignIn extends React.Component {
   // uses setLocalToken to set token to local storage when button is clicked
-  return (
-    <form className="shadow bg-white pa2" action="./add-item">
-      <h1 className="b f1">Welcome to your smart shopping list!</h1>
-      <p className="f3">Tap “Create shopping list” to get started.</p>
-      <button onClick={setLocalToken} className="bg-green ph2 pv1 white f2 b">
-        Create shopping list
-      </button>
-      <p className="f5 gray">
-        You can also{' '}
-        <a className="black" href="/">
-          join an existing shopping list
-        </a>
-        .
-      </p>
-    </form>
-  );
+  handleClickSetLocalToken = () => {
+    setLocalToken();
+  };
+  render() {
+    return (
+      <form className="shadow bg-white pa2" action="./add-item">
+        <h1 className="b f1">Welcome to your smart shopping list!</h1>
+        <p className="f3">Tap “Create shopping list” to get started.</p>
+        <button
+          onClick={this.handleClickSetLocalToken}
+          className="bg-green ph2 pv1 white f2 b"
+        >
+          Create shopping list
+        </button>
+        <p className="f5 gray">
+          You can also{' '}
+          <a className="black" href="/join-existing">
```

Comment:
Because this is a vanilla `<a>` tag and not a [`<Link>` component](https://reacttraining.com/react-router/web/api/Link), this redirect is a hard redirect that loads the app again, and then we run into [the netlify redirect issue](https://the-collab-lab.slack.com/archives/C012AAC9D9S/p1590277044010300) that we haven't resolved yet. I'm having trouble getting to this page as a result.

Before merging, could you change this to `<Link>` instead of `<a>`?